lib/inferno/dsl/fhir_evaluator/version.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+module FhirEvaluator
```

Comment:
It should not have its own version.

Reply:
Addressed https://github.com/inferno-framework/inferno-core/pull/557/commits/b73eaf3a9fa5480c3f8f02c365c9645e003a86ac